scalac-scoverage-plugin/src/main/scala/scoverage/report/ScoverageXmlWriter.scala

```diff
@@ -37,7 +37,7 @@ class ScoverageXmlWriter(sourceDirectories: Seq[File], outputDir: File, debug: B
         <statement package={stmt.location.packageName}
                    class={stmt.location.className}
                    class-type={stmt.location.classType.toString}
-                   top-level-class={stmt.location.topLevelClass}
+                   full-class-name={stmt.location.fullClassName}
```

Comment:
I believe that the XML format used here is internal to `scoverage`, so can be changed without BC concerns